### PR TITLE
Update yunexiz.json A records

### PR DESCRIPTION
**Pull request by Yunexiz**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/yunexiz.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "email": "yunexiz@aol.com"
     },
     "record": {
-        "A": ["45.137.70.68"]
+        "A": ["109.71.252.37"]
```
Comment (Stef-00012) [inline domains/yunexiz.json:7]: I cannot reach your website (500 internal server error)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/yunexiz.json (hunk 1 above)
Comment (Yunexiz) [reply domains/yunexiz.json:7]: the ip is not a single server ip, (if thats what you were thinking) its a proxy

Comment (Yunexiz) [reply domains/yunexiz.json:7]: for the website its bali.weissx.host:14104 but im trying to proxy yunexiz.is-a.dev on it

Comment (gunter228) [reply domains/yunexiz.json:7]: try this link: 109.71.252.37:14104

Comment (AlmaziikDev) [reply domains/yunexiz.json:7]: try this link: 109.71.252.37:14104

**Review by iostpa — APPROVED** [review]

**Pull request merged**